### PR TITLE
A few more variscite changes.

### PR DESCRIPTION
**Pull request by drewmoseley**

### CODE REVIEW

**Comment by drewmoseley** [comment]
Still need to replicate the above changes for the var-dart.

File: meta-mender-variscite/recipes-bsp/u-boot/patches/0001-mx6var_som-Find-splash-screen-with-Mender.patch [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
++	sd_part = emmc_part = getenv_ulong("mender_boot_part", 10, 0);
+ 
+ 	sprintf(sd_devpart_str, "0:%d", sd_part);
+ 	sprintf(emmc_devpart_str, "1:%d", emmc_part);
+@@ -238,7 +238,7 @@ int splash_screen_prepare(void)
+ 			.storage = SPLASH_STORAGE_NAND,
+ 			.flags = SPLASH_STORAGE_FS,
+ 			.mtdpart = "rootfs",
+-			.ubivol = "ubi0:rootfs",
++			.ubivol = "ubi0:rootfsa",
```
[the remark this answers is not in the document]
Comment (drewmoseley) [reply meta-mender-variscite/recipes-bsp/u-boot/patches/0001-mx6var_som-Find-splash-screen-with-Mender.patch]: Probably but I don't have nand working yet to test.  Once I get there I'll see if I can do that.

[the remark this answers is not in the document]
Comment (drewmoseley) [reply meta-mender-variscite/recipes-bsp/u-boot/patches/0001-mx6var_som-Find-splash-screen-with-Mender.patch]: We can wait for that and the DART support as well before merging.

**Comment by drewmoseley** [comment]
This now supports both var-som-solo and imx6ul-var-dart. 
This also supports eMMC on var-som-solo.
Adding NAND on var-som-solo next.

**Comment by mirzak** [comment]
> Adding NAND on var-som-solo next.

Do you want hold merging this until then?

**Comment by drewmoseley** [comment]
No need to wait. I don't know when I'll get back to that.

**Comment by drewmoseley** [comment]
Shouldn't you be doing anything _but_ maintenance at this time of night? ;-)

**Comment by mirzak** [comment]
This is when you do maintenance :)

**Pull request merged**